src/main/java/racingcar/service/RacingGameService.java

```diff
@@ -0,0 +1,50 @@
+package racingcar.service;
+
+import camp.nextstep.edu.missionutils.Randoms;
+import racingcar.domain.Car;
+import racingcar.validation.CarNameValidator;
+import racingcar.validation.MoveCountValidator;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class RacingGameService {
+    public List<Car> createCars(String carNamesInput) {
+        List<String> carNames = parseAndValidateCarNames(carNamesInput);
+        return carNames.stream()
+                .map(Car::new)
+                .toList();
+    }
+
+    public int validateAndParseMoveCount(String moveCountInput) {
+        MoveCountValidator.validateMoveCount(moveCountInput);
+        return Integer.parseInt(moveCountInput);
+    }
```

Comment:
클래스 이름이 _moveCount_ 를 포함하고 있으니 검증 메서드 이름은 좀더 간략하게 지어도 의미가 충분히 전달될 것 같습니다 !

```suggestion
    public int validateAndParseMoveCount(String moveCountInput) {
        MoveCountValidator.validate(moveCountInput);
        return Integer.parseInt(moveCountInput);
    }
```

Reply:
초기에는 자동차 이름 검증 로직과 이동 횟수 검증 로직이 한 클래스에 구현하여서 분리하는 리팩토링을 진행했는데, 그 과정에서 메서드명도 좀 더 신경 써야겠네요. 감사합니다!